api/src/main/java/org/apache/any23/vocab/HCard.java

```diff
@@ -36,10 +36,9 @@ public static HCard getInstance() {
         return instance;
     }
 
-    public IRI Card  = createClass(NS, "Card");
-    public IRI Address   = createClass(NS, "Address");
-    public IRI Geo = createClass(NS, "Geo");
-
+    public IRI card  = createClass(NS, "Card");
+    public IRI address   = createClass(NS, "Address");
+    public IRI _Geo = createClass(NS, "Geo");
```

Comment:
These are breaking changes. We probably don't want to release another major version for this, so if we want to change the naming, we should probably deprecate the old ones rather than removing them.

But in any case, how is `_Geo` an improvement over `Geo`? Not sure I agree with that naming change...

Reply:
By "breaking change", simply I mean a backwards-incompatible change (see https://semver.org/ for more info).

I would say, it's probably a good idea to disable the capitalization check for now, since so many parts of our codebase are already written the way they are.

Reply:
Reverted this and the rest and disabled check. I agree with you.